packages/gatsby/cache-dir/__tests__/static-entry.js

```diff
@@ -61,6 +61,48 @@ const reverseHeadersPlugin = {
   },
 }
 
+const setNullHeaderPlugin = {
+  plugin: {
+    onPreRenderHTML: ({ replaceHeadComponents }) => {
+      replaceHeadComponents(null)
+    },
+  },
+}
+
+const setNullHeadersPlugin = {
+  plugin: {
+    onPreRenderHTML: ({ replaceHeadComponents }) => {
+      replaceHeadComponents([null, null])
+    },
+  },
+}
+
+const setEmptyArrayHeaderPlugin = {
+  plugin: {
+    onPreRenderHTML: ({ replaceHeadComponents }) => {
+      replaceHeadComponents([])
+    },
+  },
+}
+
+const setEmptyArrayHeadersPlugin = {
+  plugin: {
+    onPreRenderHTML: ({ replaceHeadComponents }) => {
+      replaceHeadComponents([[], []])
+    },
+  },
+}
+
+const checkNonEmptyHeadersPlugin = {
+  plugin: {
+    onPreRenderHTML: ({ getHeadComponents }) => {
+      const headComponents = getHeadComponents()
+      expect(headComponents.includes(null)).toBeFalsy()
+      expect(headComponents.find(val => Array.isArray(val) && val.length === 0))
```

Comment:
this seems to miss assertion? the `expect()` should be falsy here as well, right?

Reply:
Good catch! It's a midnight-coding error on my part.